subsys/net/lib/http/http_client.c

```diff
@@ -29,12 +29,30 @@ LOG_MODULE_REGISTER(net_http_client, CONFIG_NET_HTTP_LOG_LEVEL);
 #define HTTP_CONTENT_LEN_SIZE 11
 #define MAX_SEND_BUF_LEN 192
 
-static int sendall(int sock, const void *buf, size_t len)
+static int sendall(int sock, const void *buf, size_t len,
+			const k_timepoint_t req_end_timepoint)
```

Comment:
Alignment of some functions still off...

Reply:
@pdgendt 
I couldn't find any clear rules in Zephyr Contributing quidelines on indentation.
De facto, though, indentation should be done with tabs (otherwise CI throws an error).
So here just with tabs you can't always align next line of parameters properly.

So if you have a suggestion here (apart from using spaces), I'm open for a change

Reply:
> De facto, though, indentation should be done with tabs (otherwise CI throws an error).
So here just with tabs you can't always align next line of parameters properly.

Finetuning indentation with spaces does not throw an error (I. e. use spaces instead of the last tab in the line). Sure it does not seem to be covered by the coding guidelines, but we tend to align to the opening brace all over the codebase, it simply looks neater.

Reply:
So you have to indent with tabs, and align with the opening brace using spaces.

Reply:
Okay, will do!

Reply:
Too late, it got merged... But I'll keep in mind for the next PR, thanks!